packages/stryker-jest-runner/src/jestTestAdapters/JestTestAdapter.ts

```diff
@@ -1,5 +1,5 @@
 import { RunResult } from 'jest';
 
 export default interface JestTestAdapter {
-  run(config: object, projectRoot: string): Promise<RunResult>;
+  run(config: object, projectRoot: string, enableFindRelatedTests: boolean, mutatedFileName?: string): Promise<RunResult>;
```

Comment:
Could we leave the name `mutatedFileName` out of this interface. It is not a clean separation of concerns. The JestTestAdapter shouldn't care about mutants, only about running tests. I think these 2 options can be combined into 1: `fileNameUnderTest` or something. If it is provided, it can add `{findRelatedTests: true, _: [fileNameUnderTest]`.

Reply:
I'm not sure I understand. Do you mean to only pass the `fileNameUnderTest` when running related to a specific file? I.e. when the flag is true and we're testing a mutant?